extensions/reviewed/Gamepads.json

```diff
@@ -1672,6 +1841,48 @@
         }
       ],
       "objectGroups": []
+    },
+    {
+      "description": "Return the strength of the weak vibration motor on the gamepad.",
+      "fullName": "Weak rumble magnitude",
+      "functionType": "Expression",
+      "name": "WeakVibrationMagnitude",
+      "sentence": "",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "eventsFunctionContext.returnValue = gdjs._extensionController.rumble.weakMagnitude;",
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "expressionType": {
+        "type": "expression"
+      },
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Return the strength of the strong vibration motor on the gamepad.",
+      "fullName": "Strong rumble magnitude",
+      "functionType": "Expression",
+      "name": "StrongVibrationMagnitude",
+      "sentence": "",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "eventsFunctionContext.returnValue = gdjs._extensionController.rumble.strongMagnitude;",
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "expressionType": {
+        "type": "expression"
+      },
+      "parameters": [],
+      "objectGroups": []
     }
```

Comment:
Why is this expression shared by all gamepad ?

Reply:
I think there is no need to make these values unique for each players.